STACpopulator/stac_utils.py

```diff
@@ -0,0 +1,65 @@
+import os
+from datetime import datetime
+from typing import Any
+
+import pystac
+import requests
+
+
+def stac_collection_exists(stac_host: str, collection_id: str):
+    """
+    Get a STAC collection
+
+    Returns the collection JSON.
+    """
+    r = requests.get(os.path.join(stac_host, "collections", collection_id), verify=False)
+
+    return True if r.status_code == 200 else False
```

Comment:
can be simplified to `return r.status_code == 200`.

Reply:
Right, of course....